implementations.md

```diff
@@ -19,3 +19,25 @@ If you know of any associated projects that are not listed here, please file a p
 ## Testing
 
 * [huawei-openlab/oct](https://github.com/huawei-openlab/oct) - Open Container Testing framework for OCI configuration and runtime
+
+## Hooks
+
+For addressing container issues that are outside the scope of this
```

Comment:
I wonder what became with `To keep consistency throughout the Markdown files in the Open Container spec all files should be formatted one sentence per line.` I personally hate that rule, but I know that you love it :)